docs/core-concepts/core-sdk/cli/dotnet-build.md

```diff
@@ -0,0 +1,85 @@
+dotnet-build
+===========
+
+## NAME 
+dotnet-build -- builds a project and all of its' dependencies 
+
+## SYNOPSIS
+
+dotnet-build [--output]  
+    [--build-base-path] [--framework]  
+    [--configuration]  [--runtime] [--version-suffix]
+    [--build-profile]  [--no-incremental] [--no-dependencies]
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-build` builds multiple source file from a source project and its dependencies into a binary. 
+The binary will be in Intermmidiate Language (IL) by default and will have a DLL extension. 
+`dotnet-build` will also drop a \*.deps file which outlines what the runner needs to run the application.  
+
+Building requires an existence of a lock file which means that a `dotnet-restore` call needs to happen 
+previous to building.
+
+Before any compilation begins, the build verb analyzes the project and its dependencies for incremental safety checks. 
+If all checks clear out, then build proceeds with incremental compilation of the project and its dependencies; 
```

Comment:
nit: missing comma after otherwise
"If all checks clear out": this sentence is correct but hard to read. It took me a second to realize check here was the subject and not the verb. Maybe add the word safety to make it easier to follow.